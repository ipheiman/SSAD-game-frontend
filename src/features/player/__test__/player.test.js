import React, { Component } from "react";
import ReactDOM from "react-dom";
import Player from "../../Player";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import renderer from "react-test-renderer";

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Player />, div);
});

it("should take a snapshot", () => {
  const { asFragment } = render(<Player />);

  expect(asFragment(<Player />)).toMatchSnapshot();
});
