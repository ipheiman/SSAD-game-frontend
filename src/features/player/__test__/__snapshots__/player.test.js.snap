// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    style="position: absolute; top: 0px; left: 0px; background-image: url(rsz_pokemonplayer.png); background-position: 0px 0px; width: 40px; height: 40px;"
  />
</DocumentFragment>
`;
